jib-plugins-common/src/test/java/com/google/cloud/tools/jib/api/HttpGetVerifier.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.api;
+
+import com.google.cloud.tools.jib.blob.Blobs;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import javax.annotation.Nullable;
+import org.junit.Assert;
+
+/** Verifies the response of HTTP GET. */
+public class HttpGetVerifier {
+
+  /**
+   * Verifies the response body. Repeatedly tries {@code url} at the interval of .5 seconds for up
+   * to 20 seconds until getting OK HTTP response code.
+   */
+  public static void verifyBody(String expectedBody, URL url) throws InterruptedException {
+    Assert.assertEquals(expectedBody, getContent(url));
+  }
+
+  @Nullable
+  public static String getContent(URL url) throws InterruptedException {
```

Comment:
Do we need this to be public?

Reply:
Oh true, this can be kept private now that the individual tests have been updated to use `verifyBody()` for making assertions. Updated!